frontend/packages/console-app/src/components/modals/resource-limits/ResourceLimitsModal.tsx

```diff
@@ -0,0 +1,43 @@
+import * as _ from 'lodash';
+import { FormikProps, FormikValues } from 'formik';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+} from '@console/internal/components/factory/modal';
+import ResourceLimitSection from '@console/dev-console/src/components/import/advanced/ResourceLimitSection';
```

Comment:
I know its a hassle right now since we do not have an eslint rule for it yet but try to sort the imports wherever possible.